.github/workflows/ci.yml

```diff
@@ -50,28 +50,50 @@ jobs:
         id: vars
         shell: bash
         run: |
-          echo "FINK_K8S_IMAGE=$(. ./conf.sh && echo $FINK_K8S_IMAGE)" >> $GITHUB_ENV
-          echo "FINK_K8S_IMAGE=${FINK_K8S_IMAGE}"
-      - name: Install pre-requisites (spark)
-        run: ./bin/prereq-install.sh
-      - name: Start minikube
+          echo "IMAGE=$(. ./conf.sh && echo $IMAGE)" >> $GITHUB_ENV
+          echo "IMAGE=${IMAGE}"
+          echo "FINK_ALERT_SIMULATOR_DIR=$(. ./conf.sh && echo $FINK_ALERT_SIMULATOR_DIR)" >> $GITHUB_ENV
+      - name: Clone fink-alert-simulator code
+        run: ./bin/clone-fink-alert-simulator.sh
+      - name: Create k8s/kind cluster
+        run: |
+          git clone --depth 1 -b "k8s-1.25.0" --single-branch https://github.com/k8s-school/kind-helper.git
```

Comment:
Is `k8s-1.25.0` a version number that will change frequently? Should we put it in a env variable instead?